wp-parsely.php

```diff
@@ -339,13 +355,30 @@ public function insert_parsely_page() {
             $tags = array_unique($tags);
 
             $parselyPage['@type']          = 'NewsArticle';
+            $parselyPage['mainEntityOfPage'] = array(
+                '@type' => 'WebPage',
+                '@id' => get_permalink()
+            );
             $parselyPage['headline']       = $this->get_clean_parsely_page_value(get_the_title());
             $parselyPage['url']            = get_permalink();
             $parselyPage['thumbnailUrl']   = $image_url;
-            $parselyPage['articleId']      = $postId;
+            $parselyPage['image']          = array(
+                '@type' => 'ImageObject',
+                'url' => $image_url
+            );
             $parselyPage['dateCreated']    = gmdate('Y-m-d\TH:i:s\Z', get_post_time('U', true));
+            $parselyPage['datePublished']  = gmdate('Y-m-d\TH:i:s\Z', get_post_time('U', true));
+            $parselyPage['dateModified']   = gmdate('Y-m-d\TH:i:s\Z', get_the_modified_date('U', true));
             $parselyPage['articleSection'] = $category;
+             $parselyPage['author']        = array(
+                '@type' => 'Person',
+                'name' => $authors
```

Comment:
how does Google want this expressed? should there be a new person object for each author? if so you'll have to expand the array here to have a new type = Person for each author